lib/payment/merchant_stripe.rb

```diff
@@ -179,4 +179,72 @@ def fields_needed
         .join(', ')
     end
   end
+
+  # Representation of a {Charge} object in the Payment Service.
+  class Charge < StripeHash
+    include Comparable
+    def <=>(other)
+      charge_id <=> other.charge_id
+    end
+
+    attr_accessor :charge_id, :status, :amount_cents, :currency,
+                  :customer_name, :credit_card_brand, :created_at
+
+    # Fetch the list of {Charge}s for the given {Merchant} UUID from the Payment
+    #   Service. {Charge} objects are in reverse chronological order according
+    #   to their {#created_at} attribute.
+    #
+    # @todo The complete list of charges might be very long. Therefore, we have
+    #   to add a way to paginate the list.
+    #
+    # @param merchant_id [String] UUID.
+    # @return [Array<MerchantProfile::Charge>]
+    def self.all(merchant_id)
+      connector = OrganizationConnector.new(merchant_id)
+      connector.get_charges({}).map { |charge_attrs| new(charge_attrs) }
+    end
+  end
+
+  # Representation of a {Payment} object in the Payment Service.
+  class MerchantPayment < StripeHash
+    attr_accessor :id, :meta, :stripe, :stripe_publishable_key
+
+    class << self
+      def from_tss(profile_id)
```

Comment:
Could we do the renaming (tss -> payment) in this PR? If not, we could do another one later.